plugin/config.json

```diff
@@ -3,5 +3,5 @@
     "apiLogPath": null,
     "webBindAddress": "127.0.0.1",
     "webBindPort": 8765,
-    "webCorsOrigin": "http://localhost"
+    "webCorsOrigin": ["http://localhost"]
```

Comment:
Does changing the type of the setting value work fine for people who have installed the previous version? May be better to retire the current key and instead create one called webCorsOriginList or something like that.

Reply:
Yes, you're are right. I'll rename it accordingly. Do you propose to remove the old key immediately or just mark it as deprecated and keep the support until it will be removed in the future?

Reply:
I have renamed the field to 'webCorsOriginList' like you proposed but kept support for the original 'webCorsOrigin' field (for now). I would like to remove it completely, because this leads to some unnecessary configuration redundancy. Do you see any related problems that might occur?